Wikipedia/assets/codemirror/codemirror-index.html

```diff
@@ -372,13 +374,16 @@
     }
         
     const applyTheme = (themeName) => {
-      document.getElementById('codemirror-theme').setAttribute('href', `codemirror-${themeName}.css`)  
+      document.body.className = `cm-mw-${themeName}`  
```

Comment:
Would this be a good time to check if the built-in them switching mechanism is more performant? 
`editor.setOption("theme", theme)`

Reply:
This is the same approach as `setOption("theme", theme)` but also handles setting the background color of the body to prevent any color flash on startup